CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.32.7] - 2024-10-15
+
+### Fixed
+- Remove the ability to read VCS password from a file which is no longer necessary.
```

Comment:
Might clarify that this applies to `sat bootprep`. Does it apply anywhere else?

Reply:
Even I think only on `sat bootprep`, I'll mention that. Thanks!